Appraisals

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
prefer using wwtd and manually maintained gem files ... keep the previous structure 🤷

Reply:
Oh cool! I missed it 🙈